src/sentry/integrations/vercel/uninstall.py

```diff
@@ -22,12 +109,38 @@ def post(self, request):
         # webhook event, in the future will need to check the "type" attribute
         # when handling more webhook event types
         # https://vercel.com/docs/integrations?query=event%20paylo#webhooks/events
+        if not request.META.get("HTTP_X_VERCEL_SIGNATURE"):
```

Comment:
side note: they send both `HTTP_X_VERCEL_SIGNATURE` and `HTTP_X_ZEIT_SIGNATURE` but we should use the vercel one